apiserver/facades/client/applicationoffers/base.go

```diff
@@ -492,7 +492,7 @@ func (api *BaseAPI) spacesAndBindingParams(
 		if bindings == nil {
 			bindings = make(map[string]string)
 		}
-		bindings[ep.Name] = network.DefaultSpaceName
+		bindings[ep.Name] = network.AlphaSpaceName
```

Comment:
I am not sure about this one, but I have a feeling "" is what we want.
@achilleasa ?

Reply:
Yes, we want "".  When looking at this code, I discovered a bug in that we were comparing space ids across models, which is incorrect.  Changed so we compare space names.  Any new bindings are set to be the default endpoint value if that's available, otherwise "".